index.html

```diff
@@ -1,5 +1,6 @@
 <html>
 <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
This helps adjust most everything. The changes to CSS, insure everything else fits in the box. like the comment on <b>pre</b>

This will adjust the scale of the text, but <b>pre</b> will still extend past the intend frame:
![example](https://cloud.githubusercontent.com/assets/5866400/12274941/8c50ab70-b92a-11e5-8542-8d973102fb46.png)